addons/event/models/event_event.py

```diff
@@ -88,6 +90,16 @@ class EventEvent(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
     _order = 'date_begin'
 
+    @api.model
+    def default_get(self, default_fields):
+        result = super().default_get(default_fields)
+        if not result.get('date_begin'):
```

Comment:
You are supposed to also check that "date_begin" is in the list of requested fields.

So `if 'date_begin' in default_fields and 'date_begin' not in result`.

Also a bit better to check for the key existence rather than the Falsy value, that way we allow potential overrides to give a Falsy value to it based on some conditions.

(Same logic for 'date_end', but you will also have to check that date_begin is not Falsy)